### DISCUSSION TITLE
Not booting without HDMI

### DISCUSSION THREAD

**Discussion by cleggers21** [procount/pinn#846] [Q&A]
I thought I was winning having got 2 x RPi OSes running on what is my Pi5 AllSky camera. I've got remote access through SSH and Pi Connect but as soon as I fitted the Pi on the roof - nothing, not even showing on my router.
I'm guessing its something to do with there being no HDMI connected and the config(s) need editing but I'm afraid if I start making changes I will loose all visabilty of PINN and have to start all over again.  Currently hdmi_force_hotplug = 1 which I belive makes  PINN think there is a connection even without one. Do I need to set a dummy HDMI settings in the config of each Pi OS ?
Can anyone help with what config changes should be made for remote running please.
EDIT >>> Oops - "RTFI" I should have done the remote installation first which frankly gives me the willies! Is there anyway I can rescure what I've done so far?

**Comment by procount** [comment]
See #825 
In pinn's cmdline.txt, add `video=HDMI-A-1:1920x1080M@60D`. The 'D'  is the important bit that forces hotplug. Choose an appropriate resolution.

**Comment by cleggers21** [comment]
@procount thanks, that sorted.
